src/lesson_6/script.js

```diff
@@ -0,0 +1,104 @@
+function minimum (a, b, c, d) {
+  if(! Number.isInteger(a) || !Number.isInteger(b) || !Number.isInteger(c) || !Number.isInteger(d)) {
+      console.log('input parameters are invalid');
+      return;
+  }   
+
+  let minVal = a;
+  if (minVal > b) {
+    minVal = b;
+  } 
+    if (minVal > c) {
+    minVal = c;
+  } 
+    if (minVal > d) {
+    minVal = d;
+  }
+
+  return minVal;
+} 
+
+function minDivider(m, n) {	
```

Comment:
```suggestion
function maxDivider(m, n) {	
```
Based on the task `Написати функцію, яка приймає 2 аргументи та шукає найбільший спільний дільник для них.`
It should be `max` in the function name